src/docs/common_tasks/file_bundles.md

```diff
@@ -13,15 +13,16 @@ Add a `bundles` list to your `jvm_app` target definition (more info on `jvm_app`
 Here is an example `jvm_app` definition that shows different possibilities for defining file bundles (and explains them below):
 
     ::python
-    jvm_app(name='bundle',
-      bundle(fileset=rglobs('src/main/resources/*')), # 1
-      bundle(fileset=globs('config/*') - globs('*.xml'), # 2
-      bundle(relative_to='src', fileset=globs('src/scripts/*.sh')) # 3
+    jvm_app(
+      name='bundle',
+      bundle(fileset=['src/main/resources/**/*')), # 1
+      bundle(fileset=['config/*', '!*.xml']), # 2
+      bundle(fileset=['src/scripts/*.sh'], relative_to='src') # 3
     )
 
 
-1. The `rglobs` definition will recursively include all files in the `src/main/resources` directory, *including all sub-directories*. More info can be found in [[Use globs and rglobs to group files|pants('src/docs/common_tasks:globs')]].
```

Comment:
We could keep pointing to `globs.md`, but I don't think it says anything new compared to here.